warrior/Actions/CommonActions/common_actions.py

```diff
@@ -118,7 +118,7 @@ def verify_resp_data(self, resp_ref, resp_pat, object_key):
         return status
 
     def store_in_repo(self, datavar=None, datavalue=None, type='str',
-                      filepath=None, jsonkey="repo_variables"):
+                      filepath=None, jsonkey="repo_variables", slurp=False):
```

Comment:
This function is about 100 lines long. Please refactor the code to reduce the length and make the code cleaner.

Reply:
I have added 5 lines of code to suffice the user requirement without breaking any existing functionality. refactoring code we can take it as different task.